source/clear-linux/guides/tooling/swupd-guide.rst

```diff
@@ -41,44 +41,41 @@ versioning usually involves:
 -  Listing and tracking every package available and installed on the
    system, none of which are directly tied to the current OS release.
 
-This can be done effectively, but given the nearly endless combinations of
```

Comment:
The padding of "This can be effectively" was added intentionally. The wording of this section was raised as a point of contention by external readers before.